olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1588832683/eclipse-che-preview-kubernetes.v9.9.9-nightly.1588832683.clusterserviceversion.yaml.diff

```diff
@@ -0,0 +1,60 @@
++++ /home/tolusha/gocode/src/github.com/eclipse/che-operator/olm/eclipse-che-preview-kubernetes/deploy/olm-catalog/eclipse-che-preview-kubernetes/9.9.9-nightly.1588832683/eclipse-che-preview-kubernetes.v9.9.9-nightly.1588832683.clusterserviceversion.yaml	2020-05-07 09:24:43.847451290 +0300
+@@ -48,13 +48,13 @@
+     capabilities: Seamless Upgrades
+     categories: Developer Tools
+     certified: "false"
+-    containerImage: quay.io/eclipse/che-operator:nightly
+-    createdAt: "2020-05-01T07:48:15Z"
++    containerImage: quay.io/eclipse/che-operator:7.12.2
++    createdAt: "2020-05-07T06:24:43Z"
+     description: A Kube-native development solution that delivers portable and collaborative
+       developer workspaces.
+     repository: https://github.com/eclipse/che-operator
+     support: Eclipse Foundation
+-  name: eclipse-che-preview-kubernetes.v9.9.9-nightly.1588319295
++  name: eclipse-che-preview-kubernetes.v9.9.9-nightly.1588832683
+   namespace: placeholder
+ spec:
+   apiservicedefinitions: {}
+@@ -248,26 +248,26 @@
+                 - name: OPERATOR_NAME
+                   value: che-operator
+                 - name: CHE_VERSION
+-                  value: 7.12.1
++                  value: 7.12.2
+                 - name: IMAGE_default_che_server
+-                  value: quay.io/eclipse/che-server:7.12.1
++                  value: quay.io/eclipse/che-server:7.12.2
+                 - name: IMAGE_default_plugin_registry
+-                  value: quay.io/eclipse/che-plugin-registry:7.12.1
++                  value: quay.io/eclipse/che-plugin-registry:7.12.2
+                 - name: IMAGE_default_devfile_registry
+-                  value: quay.io/eclipse/che-devfile-registry:7.12.1
++                  value: quay.io/eclipse/che-devfile-registry:7.12.2
+                 - name: IMAGE_default_pvc_jobs
+                   value: registry.access.redhat.com/ubi8-minimal:8.2-267
+                 - name: IMAGE_default_postgres
+                   value: centos/postgresql-96-centos7:9.6
+                 - name: IMAGE_default_keycloak
+-                  value: quay.io/eclipse/che-keycloak:7.12.1
++                  value: quay.io/eclipse/che-keycloak:7.12.2
+                 - name: IMAGE_default_che_workspace_plugin_broker_metadata
+                   value: quay.io/eclipse/che-plugin-metadata-broker:v3.1.2
+                 - name: IMAGE_default_che_workspace_plugin_broker_artifacts
+                   value: quay.io/eclipse/che-plugin-artifacts-broker:v3.1.2
+                 - name: IMAGE_default_che_server_secure_exposer_jwt_proxy_image
+                   value: quay.io/eclipse/che-jwtproxy:fd94e60
+-                image: quay.io/eclipse/che-operator:nightly
++                image: quay.io/eclipse/che-operator:7.12.2
```

Comment:
@mmorhun This seems wrong in a nightly CSV no ?

I assume this should be fixed on master if we still want to have the nightly channel of the preview package effectively start the nightly operator.
 
cc @tolusha